WalletWasabi.Fluent/Models/Wallets/WalletCoinsModel.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using DynamicData;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Models.Wallets;
+
+[AutoInterface]
+public partial class WalletCoinsModel
+{
+	private readonly Wallet _wallet;
+
+	public WalletCoinsModel(Wallet wallet, IWalletModel walletModel)
+	{
+		_wallet = wallet;
+
+		var initialCoinList = Observable.Defer(() => GetCoins().ToObservable());
```

Comment:
Instead of `Defer()` we could use `StartWith()`, so the actual code that needs to be executed won't be duplicated.

Reply:
Do you mean this?

```
		Pockets = signals
			.SelectMany(_ => GetPockets())
			.StartWith(GetPockets())
			.ToObservableChangeSet(x => x.Labels);
```

Reply:
> Do you mean this?
> 
> ```
> 		Pockets = signals
> 			.SelectMany(_ => GetPockets())
> 			.StartWith(GetPockets())
> 			.ToObservableChangeSet(x => x.Labels);
> ```

No I mean this:

 ```c#
Pockets = signals
	.StartWith(Unit.Default)
	.SelectMany(_ => GetPockets())
	.ToObservableChangeSet(x => x.Labels);
 ```
 
 So `GetPockets()` is not duplicated.

Reply:
Yes, that's way nicer :)